test_that("Convertion works", {
  ## Should write exception test here
})
